src/features/list-view/styles/ListView.scss

```diff
@@ -24,13 +24,16 @@
             cursor: pointer;
         }
 
-        .bdl-icon-sort-chevron.bdl-ListView-isSortAsc {
+        .bdl-icon-sort-chevron {
```

Comment:
Added this change in this PR since I didn't think we need a separate PR just for a small change. Will link this PR with the relevant ticket that is related to the icon being 12px by 12px.